draft-vvv-webtransport-overview.md

```diff
@@ -139,8 +147,8 @@ Message:
 Transport feature:
 
 : A transport feature refers to the ability of a specific transport to
-  provide a specific way of communicating data, such as supporting datagrams or
-  streams.
+  provide a specific mechanism to communicate data.  An example is the ability
+  to send datagrams that are not retransmitted in case of packet loss.
```

Comment:
That's a transport property.  The feature I was trying to define here are "supports streams" and "support datagrams".

Reply:
I don't understand the difference

Reply:
It's a difference between "what you can do with transport" and "how the transport will do it".  I agree it's not clear, so I just deleted that definition from the list.